openshift/console
# OCPBUGS-42000: use TaskRuns `results.tekton.dev/record` annotation to get the logs

**Fixes**: 
https://issues.redhat.com/browse/OCPBUGS-42000

**Solution Description**: 
We were making 2 API calls to get the logs for the PipelineRuns. Now used `results.tekton.dev/record` annotation and replaced the `records` in the value of the annotation with `logs` to get the logs of the PipelineRuns.
  
**Screen shots / Gifs for design review**: 

----Pipelines 1.15.x---

https://drive.google.com/file/d/11t96sxdHTlE2lvLYRRcJMEBWLdz2Fnnn/view?usp=drive_link


----Pipelines 1.16---

https://drive.google.com/file/d/1YowjpGQ6q-2lpMLE1mCSQXH7V770ajZa/view?usp=drive_link






**Unit test coverage report**: 
NA

**Test setup:**

    1. Install Pipelines operator and install Tekton results
    2. Verify all the pages by deleting some PipelineRuns and TaskRuns
    

**Browser conformance**:
